cardano-db-sync/src/Cardano/DbSync/Config/Types.hs

```diff
@@ -150,14 +155,20 @@ data SyncPreConfig = SyncPreConfig
   }
   deriving (Show)
 
-data SyncInsertConfig
-  = FullInsertOptions
-  | OnlyUTxOInsertOptions
-  | OnlyGovInsertOptions
-  | DisableAllInsertOptions
-  | SyncInsertConfig SyncInsertOptions
+data SyncInsertConfig = SyncInsertConfig
+  { sicPreset :: Maybe Text
+  , sicOptions :: SyncInsertOptions
+  }
   deriving (Eq, Show)
 
+-- data SyncInsertConfig
+--   = FullInsertOptions
+--   | OnlyUTxOInsertOptions
+--   | OnlyGovInsertOptions
+--   | DisableAllInsertOptions
+--   | SyncInsertConfig SyncInsertOptions
+--   deriving (Eq, Show)
+
```

Comment:
What's this?

Reply:
I think instead of completely removing this, we can slightly change it:
```
data SyncInsertPreset
   = FullInsertPreset
   | OnlyUTxOInsertPreset
   | OnlyGovInsertPreset
   | DisableAllInsertPreset
  deriving (Eq, Show)
```

and then sicPreset can be `Maybe SyncInsertPreset`. This would avoid the stringly typed tests below

Reply:
sorted, not a clue what I was thinking there!! 😅